force-app/main/default/aura/disbursementsCreate/disbursementsCreateHelper.js

```diff
@@ -1,62 +1,12 @@
 ({
-    DEFAULT_MODEL: {
-        request: null,
-        formData: {
-            paymentCount: 1,
-            paymentTotal: 0,
-            firstDate: null,
-            intervalCount: 1,
-            intervalType: "Month"
-        },
-        formDefaults: {
-            intervalTypes: [
-                {
-                    value: "Week",
-                    label: "Week"
-                },
-                {
-                    value: "Month",
-                    label: "Month"
-                },
-                {
-                    value: "Year",
-                    label: "Year"
-                }
-            ],
-            columns: [
-                {
-                    label: "Amount",
-                    fieldName: "amount",
-                    type: "currency",
-                    editable: true,
-                    cellAttributes: {
-                        alignment: "left"
-                    }
-                },
-                {
-                    label: "Scheduled Date",
-                    fieldName: "scheduleDate",
-                    type: "date-local",
-                    typeAttributes: {
-                        year: "numeric",
-                        month: "numeric",
-                        day: "numeric"
-                    },
-                    editable: true,
-                    cellAttributes: {
-                        alignment: "left"
-                    }
-                }
-            ]
-        },
-        disbursements: [],
-        uiMessages: []
-    },
-
+    DEFAULT_MODEL: {},
     VIEW_MODEL: {},
 
     init: function (cmp) {
+        // Set Default Model Values
+        this.setDefaultModel(cmp);
```

Comment:
@jdavilasf : 
Can setDefaultModel be moved to the helper and called here without needed to set the component's attribute ?
Something like :
`this.DEFAULT_MODEL = JSON.parse(JSON.stringify(helper.setDefaultModel(cmp)));`

Reply:
I think we can set the `this.DEFAULT_MODEL` up directly on the `setDefaultModel` method. If that's possible we won't need the defaultModel attribute. Let me try it. Good suggestion @sella24 👍

Reply:
All set @sella24 . I updated the helper and removed the aura attribute `defaultModel`. Now we populate the `DEFAULT_MODEL` object directly from the method. I tested it again and works well. Feel free to check again. Thanks!